src/DotVVM.Framework/DotVVM.Framework.csproj

```diff
@@ -92,6 +92,11 @@
     <Reference Include="System.ComponentModel.DataAnnotations" />
     <Reference Include="Microsoft.CSharp" />
   </ItemGroup>
+  <ItemGroup>
+    <Reference Include="Microsoft.Build.Tasks.v4.0">
+      <HintPath>..\..\..\..\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.7.1\Microsoft.Build.Tasks.v4.0.dll</HintPath>
+    </Reference>
```

Comment:
What the hell is this?

Reply:
This should not be there. I didn't added it intentionally. I will remove it.